config/software/datadog-metro.rb

```diff
@@ -0,0 +1,31 @@
+name "datadog-metro"
+default_version "last-stable"
+github_token = ENV['GITHUB_TOKEN']
+
+env = {
+  "GOROOT" => "/usr/local/go",
+  "GOPATH" => "/var/cache/omnibus/src/datadog-metro"
+}
+
+dependency "libpcap"
+
+if ohai['platform_family'] == 'mac_os_x'
+  env.delete "GOROOT"
+  gobin = "/usr/local/bin/go"
+else
+  gobin = "/usr/local/go/bin/go"
+end
+
+build do
+   ship_license "https://raw.githubusercontent.com/DataDog/go-metro/master/LICENSE"
+   command "mkdir -p /var/cache/omnibus/src/datadog-metro/src/github.com/DataDog", :env => env
+   command "git clone https://#{github_token}:x-oauth-basic@github.com/DataDog/go-metro.git", :env => env, :cwd => "/var/cache/omnibus/src/datadog-metro/src/github.com/DataDog"
```

Comment:
You should clone the repo only if the directory does not exist, otherwise the `clone` will fail.